nextroute/model_expression_custom.go

```diff
@@ -136,6 +155,19 @@ type VehicleTypeExpression interface {
 	)
 }
 
+// VehicleTypeDistanceExpression is a distance expression that has a value for
+// each vehicle type.
+type VehicleTypeDistanceExpression interface {
+	VehicleExpression
```

Comment:
This can be a VehicleTypeExpression (something that is dependent on the type of vehicle) or cleaner it can be a DefaultExpression